packages/connection-form/src/components/advanced-options-tabs/csfle-tab/encrypted-field-config-input.tsx

```diff
@@ -55,15 +36,12 @@ function EncryptedFieldConfigInput({
           Add an optional client-side EncryptedFieldConfigMap for enhanced
           security.
         </Description>
-        <AceEditor
-          mode="javascript" // will be set to mongodb as part of OPTIONS
-          theme="mongodb"
-          width="100%"
-          value={encryptedFieldConfigToText(encryptedFieldConfig)}
-          onChange={(newText) => onChange(textToEncryptedFieldConfig(newText))}
-          editorProps={{ $blockScrolling: Infinity }}
-          name="import-pipeline-editor"
-          setOptions={OPTIONS}
+        <Editor
+          variant="Shell"
```

Comment:
don't we use the enum here?

Reply:
Line 40 (sorry :))

Reply:
the enum is just so we don’t do typos in non-TS files, here TS already takes care of that for us